lib/Calendar.js

```diff
@@ -724,7 +726,15 @@ Calendar.propTypes = {
     day: _propTypes2.default.node,
     agenda: _propTypes2.default.node,
     showMore: _propTypes2.default.func
-  })
+  }),
+
+  /**
+   * Ref for the calendar's root div. Can be either the old ref style (function/string)
+   * or the new ref style (React.createRef).
+   */
+  ref: _propTypes2.default.oneOfType([_propTypes2.default.func, _propTypes2.default.string, _propTypes2.default.shape({
```

Comment:
Why do we need to worry about backwards compatibility here?

Reply:
Not sure what you mean by backwards compatibility - do you mean the old function/string refs? If so then the reason they're included is because this is a separate repo that's supposed to be as general-purpose as possible, similar to the original repo we forked. Within `hippo` we won't be using the old refs when passing into this component.